include/hipSYCL/sycl/libkernel/accessor.hpp

```diff
@@ -2250,6 +2267,11 @@ class local_accessor
       _num_elements{allocationSize}
   {}
 
+  specialized<address> getAddr() const { return _addr; }
+
+  specialized<range<dimensions>> getNumElem() const { return _num_elements; }
```

Comment:
```suggestion
  specialized<range<dimensions>> AdaptiveCpp_get_num_elem() const { return _num_elements; }
```
same here.
(an alternative to these getter functions might be to make every local accessor `friend` with every other one?)